include/deal.II/fe/mapping_fe_field.h

```diff
@@ -118,6 +119,31 @@ class MappingFEField : public Mapping<dim, spacedim>
                  const VectorType &    euler_vector,
                  const ComponentMask & mask = ComponentMask());
 
+  /**
+   * Constructor taking vectors on the multigrid levels rather than the active
+   * cells only. The vector of vectors is expected to have as many entries as
+   * there are levels in the triangulation and provide valid data on each
+   * level, i.e., be of compatible length DoFHandler::n_dofs(level). Apart
+   * from the source of the vectors, the same arguments as in the other
+   * constructor need to be provided.
+   */
+  MappingFEField(const DoFHandlerType &         euler_dof_handler,
+                 const std::vector<VectorType> &euler_vector,
+                 const ComponentMask &          mask = ComponentMask());
+
+  /**
+   * Constructor taking vectors on the multigrid levels rather than the active
+   * cells only, variant with MGLevelObject instead of std::vector. The vector
+   * of vectors is expected to have as many entries as there are levels in the
```

Comment:
this needs to be updated as it is no "vector of vectors"

Reply:
or replace the whole documentation block with "like above but with an MGLevelObject"